src/GafferScene/SceneNode.cpp

```diff
@@ -446,6 +446,31 @@ Imath::Box3f SceneNode::unionOfTransformedChildBounds( const ScenePath &path, co
 	return out->childBoundsPlug()->getValue();
 }
 
+bool SceneNode::enabled( const Gaffer::Context *context ) const
+{
+	const BoolPlug *p = enabledPlug()->source<BoolPlug>();
```

Comment:
This could return `nullptr` if the input plug is not a BoolPlug (e.g. an IntPlug or even a StringPlug).